monsteer/qt/SteeringWidget.h

```diff
@@ -40,15 +40,10 @@ class SteeringWidget : public QWidget
 public:
 
     /**
-     * @param selectionZeqSchema the ZeroEQ schema used for receiving selection
-     *        events.
-     * @param simulationZeqSchema the ZeroEQ schema used for sending stimuli to
-     *        the simulation.
+     * Constructor
      * @param parentWgt Parent widget.
      */
-    SteeringWidget( const servus::URI& selectionZeqSchema,
-                    const servus::URI& simulationZeqSchema,
-                    QWidget *parentWgt = 0 );
+    SteeringWidget( QWidget *parentWgt = 0 );
```

Comment:
Although testing GUI components is not easy and there's no test at the moment, that's one case in which you may want to have at least a constructor that can take URIs.